app.go

```diff
@@ -59,6 +59,9 @@ func createServer() (s *http.Server) {
 	//connect to monogoDB and select database
 	database.NewMongoDBClient(config.Env.MongoDBURI, config.Env.MongoDBName)
```

Comment:
What if an error occurs while trying to connect to the database? The server will go on to start and users will be getting errors, we don't want that...what we want is for the server to shutdown that way we know there's an error and we can fix it. 

Resolve this for both DB calls you have.